packages/uswds-core/src/styles/functions/color/get-system-color.scss

```diff
@@ -39,6 +39,8 @@ variable that is, itself, a list
   }
 
   $color-family: general.smart-quote($color-family);
+  // Allows function to take quoted or unquoted color grade key values
+  $color-grade: #{$color-grade};
```

Comment:
Sass interpolation transforms the value from a `number` to a `string`. This allows the function, `get-system-color()`, to take either a string or a number value for the color grade. 

Before/after:

```scss
  @debug type-of($color-grade); // Returns number
  $color-grade: #{$color-grade};
  @debug type-of($color-grade); // Returns string
```

The reason we want this change, is because we are sass color maps are now using quoted key values as recommended by [Sass documentation](https://arc.net/l/quote/prbozafo). Rather than changing every use of `get-system-color()` to use quoted values, we are able to make the function flexible and accept either quoted or un-quoted value.

If we had to recommend a way to pass a color grade, I suppose I would recommend quoted since the key it's checking is a string but updating our code would result in something like 500+ changed lines of code.

Reply:
@mahoneycm is this still needed if we're using quoted values now?

Reply:
Yes this was added because the values are quoted now. It enables us to use string or number values with the sass function.

Alternatively, we add quotes everywhere we use the `get-system-color()` functions, which would result in over 500 changed lines of code.

Reply:
Thanks for confirming!